src/Activity.js

```diff
@@ -104,6 +100,27 @@ class Activity extends UserFitness {
     return dates;
   }
 
+  returnStreak(amount, activityData) {
```

Comment:
It looks like this function is still in here even though we decided to leave it out. I would recommend going into your branch and making these changes - close these two pull requests and then once these issues are fixed then push up again with those fixes in place.

Reply:
I think some of the confusion might be that there are two PRs - I would say that this refactor is something that could be in one branch with one PR - if you have two branches with two PRs it can be confusing when things are being changed in both outside of the scope of the branch

Reply:
I'm a big fan of this refactoring. We made 6 different methods into 2 on that one. Cleaned up a lot of code. Good work guys!